GitDiffMargin.Extension/source.extension.vsixmanifest

```diff
@@ -13,16 +13,16 @@
         <!--<Tags></Tags>-->
     </Metadata>
     <Installation>
-        <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Community" />
-        <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
+        <InstallationTarget Version="[11.0,18.0)" Id="Microsoft.VisualStudio.Community" />
```

Comment:
What is version 18.0?

Reply:
I'll come back to answer this one as soon as I can

Reply:
I thought it was the case 😉